azure/functions/_abc.py

```diff
@@ -422,3 +422,32 @@ class OrchestrationContext(abc.ABC):
     @abc.abstractmethod
     def body(self) -> str:
         pass
+
+
+class SqlRow(abc.ABC):
```

Comment:
@YunchuWang @gavin-aguiar - Is there a ClassWide coverage NoQA to skip the coverage the abstract class?

Reply:
This should do it right?

`# flake8: noqa`

https://github.com/Azure/azure-functions-python-library/blob/dev/azure/functions/_thirdparty/werkzeug/_compat.py#L1

That seems like something that should be done separately though. Or is this a blocking issue for getting this PR in?

Reply:
This isn't blocking but would be good to get that change in along with the code it's checking. Lets get this PR in and we can get the NoQA later.